hack/dependencies.yml

```diff
@@ -1,57 +1,57 @@
-      amd64: fa954cf4e288d61e671a2be93d41d08309951d705b1f02d4c90213aa5736add5
-      arm64: b457413588f5911fd3dc04f1143fd683e322228090cb3024c0d58463ce6f0a4e
+      amd64: 1d799fe491c87c18fa3b25473ee04acceea04d410cc8f806e8a6b06fa82abaee
+      arm64: c54c5a78fde0e2db92da8afaa3d2d5b7407d291106a8b8b32af6cbde1a5b3ebc
   dev: true
   name: kbld
-  repo: vmware-tanzu/carvel-kbld
-  urlTemplate: https://github.com/vmware-tanzu/carvel-{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
-  version: v0.36.1
+  repo: carvel-dev/kbld
+  urlTemplate: https://github.com/carvel-dev/{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
+  version: v0.36.3
 - checksums:
     darwin:
-      amd64: 116a32c5974fcc08bfbec5e36b3a9145ed12457b10b0d842613add2f34c0ef03
+      amd64: 7349b265b327c6fb29fcf866e311029d3a5f06c18cb29d96b511fa378b245474
     linux:
-      amd64: 436e30a34647b2d889010d08a9242b4d3954f4c84c9ca26dd30751934ecf9950
-      arm64: 197971e92b44a54ddeb28aa3549310382c1a109e408a485b6eb41d9d0b73dba3
+      amd64: bb586f4ee94a8fdeea5b3fcb934b95b20bb0d502deab19e9a445a573b6a27071
+      arm64: 0e81eeb34bda28108f58011184c04226c422a83fc8a2d575dd5b85d4f569977a
   dev: true
   name: kapp
-  repo: vmware-tanzu/carvel-kapp
-  urlTemplate: https://github.com/vmware-tanzu/carvel-{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
-  version: v0.54.1
+  repo: carvel-dev/kapp
+  urlTemplate: https://github.com/carvel-dev/{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
+  version: v0.54.2
 - checksums:
     darwin:
-      amd64: e7fb496b8e81b4b87b0076e778a384588b8e19a7b68c2c0b2ed74739c398643f
+      amd64: 37a347de20a2f3d13b3d7f09378b9c487e03d27cab49233eb0586f7d3a974fef
     linux:
-      amd64: f5d3cbbd8135d2d48f4f007b8a933bd60b2a827d68f4001c5d1774392fa7b3f2
-      arm64: eee5abf139043c459ff2b9ffab0a75556d4930d44abc5e7e4477e84b598b6f82
+      amd64: b656312997c81dd734a3e83403b0647ceae3446366dc5f179a23f39fee1d79e7
+      arm64: ed209b7d3830f2a785b1a86807213441594f9c9d1d2ef2c52bfbb5d85a9ef569
   dev: true
   name: vendir
-  repo: vmware-tanzu/carvel-vendir
-  urlTemplate: https://github.com/vmware-tanzu/carvel-{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
-  version: v0.32.2
+  repo: carvel-dev/vendir
+  urlTemplate: https://github.com/carvel-dev/{{.Name}}/releases/download/{{.Version}}/{{.Name}}-{{.OS}}-{{.Arch}}
+  version: v0.32.4
 - checksums:
     linux:
-      amd64: 2315941a13291c277dac9f65e75ead56386440d3907e0540bf157ae70f188347
-      arm64: 57fa17b6bb040a3788116557a72579f2180ea9620b4ee8a9b7244e5901df02e4
+      amd64: 950439759ece902157cf915b209b8d694e6f675eaab5099fb7894f30eeaee9a2
+      arm64: 260cda5ff2ed5d01dd0fd6e7e09bc80126e00d8bdc55f3269d05129e32f6f99d
   dev: false
   name: helm
   repo: helm/helm
   tarballSubpath: '{{.OS}}-{{.Arch}}/helm'
   urlTemplate: https://get.helm.sh/helm-{{.Version}}-{{.OS}}-{{.Arch}}.tar.gz
-  version: v3.10.2
+  version: v3.10.3
```

Comment:
I do see that helm have made a new release last week: `3.11.0`. 

https://github.com/helm/helm/releases/tag/v3.11.0 

I believe we were looking to upgrade the helm also?

Reply:
I don't think we want to jump up a minor in a patch release. Unless we are forced to because 3.10 is out of support. Otherwise we should stick to the patches